### PR TITLE
executor: fix log format

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: executor/set.go [hunk 1]
```diff
@@ -128,7 +128,8 @@ func (e *SetExecutor) executeSet() error {
 				return errors.Trace(err)
 			}
 			e.loadSnapshotInfoSchemaIfNeeded(name)
-			log.Infof("[%d] set system variable %s = %s", sessionVars.ConnectionID, name, value.GetString())
+			_, valStr := value.ToString()
```
Comment (shenli) [inline executor/set.go]: valStr, _ := value.ToString()

**Comment by ngaut** [comment]
PTAL

**Comment by coocood** [comment]
LGTM

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**